_data/nav.yml

```diff
@@ -23,9 +23,11 @@ Components:
   - text: Overview
     href: /components/
   - href: /components/typography/
-  - href: /components/colors/
+  - text: Colors
+    href: /components/colors/
   - href: /components/icons/
-  - href: /components/grids/
+  - text: Grids
+    href: /components/grids/
```

Comment:
Another option is we could also change this to the `utilities/*` links so it's not a redirect and goes directly. Thoughts?

Reply:
I might suggest moving colors out of components altogether

Reply:
Removed "Colors" in c750d57.

Reply:
Keep grids in components?